### PR TITLE
Inventory page backend

### PR DESCRIPTION
**Pull request by gavxue**
Created a basic route builder for individual item pages

### CODE REVIEW

**Comment by gavxue** [comment]
Not sure if this is the best way to build the routes so I will need some feedback on what to change.

File: server/data-base/inventory.json [hunk 1]
```diff
@@ -5,22 +5,50 @@
   "novelties": [
     {
       "category": "Shirts and Sweaters",
-      "items": []
+      "items": [
+        {
+          "item": "Cheat Sheet Tee",
+          "price": "$20.00",
+          "description": "Available in S/M/L/XL",
+          "image": "/assets/img/inventory/novelties/shirts/cheatsheet.jpg"
```
Comment (some-molasses) [inline server/data-base/inventory.json:13]: i think we're running into the same issue of accidentally commiting images as last time - would we be able to remove the images from this PR (by moving them into the `uploads` folder)?

Comment (gavxue) [reply server/data-base/inventory.json:13]: Oh I purposely added them this time for testing purposes. Should I still remove?

Comment (some-molasses) [reply server/data-base/inventory.json:13]: oh as long as you're aware, feel free to keep for now!

File: server/data-base/inventory.json [hunk 2]
```diff
@@ -5,22 +5,50 @@
...
+      ]
     },
     {
       "category": "Ties",
-      "items": []
+      "items": [
+        {
+          "item": "2022 Pink Tie",
+          "price": "$2.00",
+          "description": null,
```
Comment (some-molasses) [inline server/data-base/inventory.json:34]: these `null`s appear to be breaking the backend - thoughts on changing them to `""`?

File: server/routes/controllers/page-loader.ts [hunk 3]
```diff
@@ -49,6 +50,24 @@ export class PageLoader {
     }
   }
 
+  static buildItemRoutes(items, router: Router) {
```
Comment (some-molasses) [inline server/routes/controllers/page-loader.ts:53]: really tiny nitpick for future readability

```suggestion
  static buildInventoryItemRoutes(items, router: Router) {
```

**Review by some-molasses — COMMENTED** [review]
looks good so far! let me know when this is finalized and alex or i can review it again!

**Pull request closed without merging**